applications/tari_console_wallet/src/ui/ui_error.rs

```diff
@@ -1,38 +0,0 @@
-// Copyright 2022 The Tari Project
-// SPDX-License-Identifier: BSD-3-Clause
-
-use tari_comms::connectivity::ConnectivityError;
-use tari_contacts::contacts_service::error::ContactsServiceError;
-use tari_utilities::hex::HexError;
-use tari_wallet::{
-    error::{WalletError, WalletStorageError},
-    output_manager_service::error::OutputManagerError,
-    transaction_service::error::TransactionServiceError,
-};
-use thiserror::Error;
-
-#[derive(Error, Debug)]
-pub enum UiError {
```

Comment:
The move of deleting this and moving it breaks the code structure away from the rest of the code base. 
And having more smaller files are easier to navigate than larger monolithic files